specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json

```diff
@@ -3648,6 +3648,240 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualHubs/{virtualHubName}/hubRouteTables/{routeTableName}": {
```

Comment:
>hubRouteTables [](start = 129, length = 14)

are there normal routeTables under virtualHubs?  Isn't it implied that a routeTable under a hub would be a "hubRouteTable"?  Not critical, just unnecessary information

Reply:
we already have an api exposed with /virtualhubs/{}/routeTables{} which we will be deprecating in the future. So we had to go with hubRouteTables in the api